### DISCUSSION TITLE
Should webots.cloud support pure geometries?

### DISCUSSION THREAD

**Discussion by BenjaminDeleze** [cyberbotics/webots-cloud#127] [General]
When going through the objects, I realized that we skipped the `/objects/geometries`, which contains the extrusion, textureBox, ...

I tried to add them, but opening them in webots.cloud result in a crash of the 3d view:
https://webots.cloud/run?version=R2023b&url=https://github.com/cyberbotics/webots/blob/develop/projects/objects/geometries/protos/Rectangle.proto

This is normal because it "wrong" to have a geometry alone (not in a shape or bounding object).

Should me make a special case to be able to display geometires in webots.cloud?

We already have such a special case to display appearances.

**Comment by omichel** [comment]
No, I believe we should leave out these geometries from webots.cloud.
